README.md

```diff
@@ -232,6 +219,42 @@ The port this server will run on.
 
 This is the `amida-auth-microservice` JWT that is used by this repo's automated test suite when it makes requests.
 
+##### `NOTIFICATION_SERVICE_MONGO_CONNECTION_STRING` (Required)
+
+Connection string that is passed to the mongo driver. This can take the generic format: `mongodb://username:password@host:port/dbname`. We recommend only specifying host, port and dbname in this string, which will look like `mongodb://host:port/dbname`. Specify the username and password using the `NOTIFICATION_SERVICE_MONGO_USER` and `NOTIFICATION_SERVICE_MONGO_PASSWORD` variables.
+- When using docker, set the host to the name of the docker container running mongo. Setting to `amida-notification-microservice-db` is recommended.
+
+##### `NOTIFICATION_SERVICE_MONGO_DB`
+
+MongoDB database name. This overrides the database name in `NOTIFICATION_SERVICE_MONGO_CONNECTION_STRING`.
+- Setting to `amida_notification_microservice` is recommended because 3rd parties could be running Amida services using their Mongo instances--which is why the name begins with `amida_`.
+
+##### `NOTIFICATION_SERVICE_MONGO_USER`
+
+Mongo user that will perform operations on behalf of this microservice. Therefore, this user must have permissions to modify the database specified by `NOTIFICATION_SERVICE_MONGO_DB` or `NOTIFICATION_SERVICE_MONGO_CONNECTION_STRING`.
+This overrides the user specified in `NOTIFICATION_SERVICE_MONGO_CONNECTION_STRING`.
+- Setting to `amida_notification_microservice` is recommended because 3rd parties could be running Amida services using their Postgres instances--which is why the name begins with `amida_`.
+
+##### `NOTIFICATION_SERVICE_MONGO_PASSWORD`
+
+Password of mongo user `NOTIFICATION_SERVICE_MONGO_USER`.
+This overrides the password specified in `NOTIFICATION_SERVICE_MONGO_CONNECTION_STRING`.
+
+##### `NOTIFICATION_SERVICE_MONGO_AUTH_SOURCE`
+
+Mongo database that should be authenticated against.
+
+##### `NOTIFICATION_SERVICE_MONGO_SSL_ENABLED` [`false`]
+
+Whether an SSL connection shall be used to connect to mongo.
+
+##### `NOTIFICATION_SERVICE_MONGO_CA_CERT`
+
+If SSL is enabled with `NOTIFICATION_SERVICE_MONGO_SSL_ENABLED` this can be set to a certificate to override the CAs that are trusted while initiating the SSL connection to postgres. Without this set, Mozilla's list of trusted CAs is used. Note that this variable should contain the certificate itself, not a filename.
```

Comment:
Tiny typo: "connection to **postgres**". Also, given this blurb was copied from the same kind of PG variable, is this blurb about overriding CAs n such still true?

Reply:
Ooops, good question - not sure. I'll look into it. If I can't find a good answer or something similar, I may just remove that blurb.

Reply:
Fixed; removed the note about Mozilla

Reply:
I updated this again; I think it's clear now